node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java

```diff
@@ -222,7 +229,7 @@ private void ensureNodeTableIsUpdated() {
 
     private void ensureClusterTableIsUpdated() {
         try {
-            if (0 == engine.getStatus(newContext().getCairoSecurityContext(), new Path(), clusterTable.name)) {
+            if (0 == engine.get().getStatus(newContext().getCairoSecurityContext(), new Path(), clusterTable.name)) {
```

Comment:
Now we need to check for null here too.

Reply:
Alternatively, just wait for the nullptr exception and fix the incorrect usage. Nothing should be using the engine after it been closed.